.github/workflows/coverage.yml

```diff
@@ -0,0 +1,62 @@
+name: Coverage
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    name: Run coverage tests
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        part: [ 1, 2 ]
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: true
+
+    - uses: niden/actions-memcached@v7
+
+    - name: Setup cache (carton)
+      uses: actions/cache@v2
+      with:
+        path: local
+        key: ${{ runner.os }}-carton-${{ hashFiles('cpanfile.snapshot') }}
+
+    - name: Setup cache (coverage)
+      uses: actions/cache@v2
+      with:
+        path: local-coverage
+        key: ${{ runner.os }}-coverage
+
+    - name: Install packages
+      run: |
+        sudo apt install -y gettext language-pack-en language-pack-de language-pack-sv libimage-magick-perl
+        vendor/bin/carton install --deployment
+        commonlib/bin/gettext-makemo FixMyStreet
+        bin/cpanm --quiet --notest -l local-coverage Devel::Cover::Report::Codecov JSON::MaybeXS
+
+    - name: Run tests (with coverage, part 1)
+      if: matrix.part == 1
+      run: |
+        script/test --jobs 3 $(find t/app/controller -name "*.t")
+      env:
+        PERL5LIB: local-coverage/lib/perl5
+        HARNESS_PERL_SWITCHES: "-MDevel::Cover=+ignore,local/lib/perl5,commonlib,perllib/Catalyst/[^A],perllib/Email,Test.pm,^t"
+
+    - name: Run tests (with coverage, part 2)
```

Comment:
Why the two parts?

Reply:
So it finishes quicker overall. It's a bit lopsided as you can see from the runs, but was at least an easy place to split them. Now it's in its own workflow, we could consider splitting further...
Originally done in f7873a97d3be13251da30f12705c3dafa5468995